commerce_coordinator/apps/commercetools/views.py

```diff
@@ -43,6 +44,8 @@ def post(self, request):
             **request.data
         }
 
+        isOrderFulfillmentForwardingEnabled = is_order_fulfillment_service_forwarding_enabled(request)
```

Comment:
we use camel case in python, sorry if I missed it in my last review

```suggestion
        is_order_fulfillment_forwarding_enabled = is_order_fulfillment_service_forwarding_enabled(request)
```

Reply:
We don't use camel case instead use snake case?